.travis.yml

```diff
@@ -18,13 +18,4 @@ notifications:
     on_failure: change
 env:
   global:
-    secure: D8bXGaTynX3VVPQtwZC03KX95EeBDxzlqhDnq6XskNcWbKwC9PBCzb/c3PF4184ra+XRf5uZZfaQjlZTlGWGfD0xCZwI6TuX0Qj2XfQT/nWhrHrFYm7oL4JeunYSIj2tv0wkIcSJL1/fsc49o+Geqf+v96HGYG1s/Sq6ig60/wQ=
-
-#deploy:
-#  provider: pages
-#  skip_cleanup: true
-#  local_dir: out
-#  on:
-#    branch: jsdoc-ghpages
-#  github-token:
-#    secure: ...
+    secure: lcdjTpuv2imFSgcYWMLV4odyZ4jOTjJhyViGg9hOFzggoPGFAjplz6bPgNJsxq20ZsFDw4UN6UlZ8lPGlByIUBhf2OXs0cEJDcNSoYAV7CuTMSdZsLDb+TrWQvjtSZMBEkeVENfJjyfiJo7PCt2/cIQXSVSnhZ5Uiir+Eus5x/s=
```

Comment:
What about [this](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings) instead using encrypted vars?

Reply:
It can be done this way, but for this purpose the encrypted tokens are perfectly fine